src/Utils/Flex/templates/cloudbuild.yaml.tmpl

```diff
@@ -0,0 +1,20 @@
+steps:
+{% if cloud_sql_instances %}
+  - name: gcr.io/cloud-builders/docker
+    args: ['pull', '{{ cloud_sql_proxy_image }}']
+    id: cloud-sql-proxy-pull
+  - name: gcr.io/cloud-builders/docker
+    args: ['run', '-d', '--network=cloudbuild', '-v', '/cloudsql:/cloudsql', '{{ cloud_sql_proxy_image }}', '/cloud_sql_proxy', '-dir=/cloudsql', '-instances={{ cloud_sql_instances }}']
+    wait_for: ['cloud-sql-proxy-pull']
+    id: cloud-sql-proxy-run
+{% endif %}
+  - name: gcr.io/cloud-builders/docker
+    args: ['pull', '{{ target_image }}']
+    id: target-image-pull
+{% if cloud_sql_instances %}
+    wait_for: ['cloud-sql-proxy-run']
+{% endif %}
+  - name: gcr.io/cloud-builders/docker
+    args: ['run', '-t', '--network=cloudbuild', '-v', '/cloudsql:/cloudsql', '{{ target_image }}', {{ commands|raw }}]
```

Comment:
Should the command being executed be quoted? Perhaps the user wants to run a command like `cd some_dir && do_something`

Reply:
These are actually already quoted. Unfortunately I don't think you can directly pass `&&` to docker run in a meaningful manner. For complex cases like that, probably you can have a shell script and run it with this tool.

Reply:
Actually you can do something like:

```
vendor/bin/flex_exec run -p -- bash -c 'cd /somewhere && do something'
```